src/Altinn.App.Core/Models/UserAction/UserActionResult.cs

```diff
@@ -1,6 +1,24 @@
-
 namespace Altinn.App.Core.Models.UserAction;
 
+/// <summary>
+/// Represents the result of a user action
+/// </summary>
+public enum ResultType
+{
+    /// <summary>
+    /// The user action succeeded
+    /// </summary>
+    Success,
+    /// <summary>
+    /// The user action failed
+    /// </summary>
+    Failure,
+    /// <summary>
+    /// The client should redirect the user to a new url
+    /// </summary>
+    Redirect
+}
+
```

Comment:
Venter vi til neste gang med å bruke innheritance for å ha flere resultat typer?

Reply:
Tja, tenkte å ikke utvide scope mer. Synes du det bør tas nå?

Reply:
Usikker. Nå endrer du `UserActionResult` på en måte som ikke er bakover kompatibel uansett, så kanskje like greit å bare gjøre det riktig.

Reply:
FYI: Lagde et draft med inheritance, men da vi så det med egne øyne så vi egentlig ikke gevinst likevel. Ble uansett en if/switch i controller, og vi har faktory-like metoder for å lage riktige resultatobjekter. Tenker det holder for nå.